knowledge_runtime/knowledge_runtime/services/duckdb_generator.py

```
@@ -0,0 +1,704 @@
+        for (table_name,) in tables:
+            try:
+                result = conn.execute(
+                    f'SELECT * FROM "{table_name}" LIMIT {sample_rows}'
+                )
+                columns = [desc[0] for desc in result.description]
+                rows = result.fetchall()
+
+                table_samples = []
+                for row in rows:
+                    row_dict = {}
+                    for col, val in zip(columns, row):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)